cypress/support/hostsfunctions.ts

```diff
@@ -239,3 +253,30 @@ export function testHostBulkDelete(host_type: string, inventory: Inventory) {
     );
   }
 }
+
+export function testMissingButton(
+  host_type: string,
+  inventory: Inventory,
+  view: string,
+  missing: string
+) {
+  //navigate to list view and check host existing
+  //in case this should be done to detailes view the only thing needed is to add click on host name
+  navigateToBaseView(host_type, inventory.name);
+  cy.get(`[aria-label="Simple table"] tr`).should('have.length', 2);
+  cy.get(missing).should('not.exist');
```

Comment:
suggestion in place of negative assertion, can we assert on what should exist instead?

Reply:
same - the element is just missing 
there is nothing there